src/base/static/components/templates/dashboard.js

```diff
@@ -0,0 +1,312 @@
+    const linechartData = Object.entries({ ...daysGrouped, ...grouped })
+      .map(([day, places]) => ({
+        date: new Date(day),
+        day,
+        count: places.length,
+      }))
+      .sort((a, b) => {
+        return a.date - b.date;
+      });
+
+    return linechartData;
+  };
+
+  render() {
+    const commentsCount =
+      this.props.places &&
+      this.props.places.reduce((count, place) => {
+        if (place.submission_sets.comments) {
+          count += place.submission_sets.comments.length;
+        }
+        return count;
+      }, 0);
+    const supportsCount =
+      this.props.places &&
+      this.props.places.reduce((count, place) => {
+        if (place.submission_sets.support) {
+          count += place.submission_sets.support.length;
+        }
+        return count;
+      }, 0);
+
+    return (
+      <DashboardWrapper>
+        <LargeTitle>{this.props.dashboardConfig.title}</LargeTitle>
+        <HorizontalRule />
+        <OverviewWrapper>
+          <RegularTitle style={{ gridArea: "title" }}>Overview</RegularTitle>
+          <Card
+            gridArea="card1"
+            label="Ideas"
+            number={this.props.places ? this.props.places.length : "..."}
+          />
+          <Card
+            gridArea="card2"
+            label="Comments"
+            number={commentsCount ? commentsCount : "..."}
+          />
+          <Card
+            gridArea="card3"
+            label="Supports"
+            number={supportsCount ? supportsCount : "..."}
+          />
+          <DownloadDataLink
+            href={`${this.props.apiRoot}${
+              this.props.dashboardConfig.datasetOwner
+            }/datasets/${
+              this.props.dashboardConfig.datasetId
+            }/mapseed-places.csv?format=csv&include_private=true&page_size=10000`}
+          >
+            {`Download Survey Data`}
+          </DownloadDataLink>
+        </OverviewWrapper>
+        <EngagementWrapper>
+          <RegularTitle>Engagement</RegularTitle>
+          <LineChart
+            width={1200}
+            height={350}
+            data={this.getLineChartData()}
+            margin={{ top: 5, right: 30, left: 20, bottom: 24 }}
+          >
+            <XAxis dataKey="day">
+              <Label value="Date" position="bottom" />
+            </XAxis>
+            <YAxis>
+              <Label value="Number of Ideas" angle={-90} />
+            </YAxis>
+            <CartesianGrid strokeDasharray="3 3" />
+            <Tooltip />
+            <Line
+              type="monotone"
+              dataKey="count"
+              stroke={BLUE}
+              activeDot={{ r: 8 }}
+            />
+          </LineChart>
+        </EngagementWrapper>
+        {this.props.dashboardConfig.surveyMetrics && (
+          <SurveyWrapper>
+            <RegularTitle style={{ gridArea: "title" }}>Survey</RegularTitle>
+            {/* TODO: Make these charts more data-driven. Right now, they are hard coded for durham */}
```

Comment:
Was this TODO addressed?

Reply:
No, but I punted it for now.

We will need to address it when we want to use the categories pie chart or the bar chart for other flavors.

Reply:
Ah, ok-- sounds good.